### PR TITLE
card was created

### PR DESCRIPTION
**Pull request by vanyachyzh**
- [DEMO LINK](https://vanyachyzh.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://vanyachyzh.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/styles/blocks/page.css [hunk 1]
```diff
@@ -0,0 +1,12 @@
+.page {
+  --blue: #00acdc;
+  font-family: Roboto, sans-serif;
```
Comment (nazarmatsevych) [inline src/styles/blocks/page.css:3]: move font-family to html selector

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,44 @@
...
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+
+        <p>Reviews: 5</p>
+      </div>
+
+      <div class="product-card__price">
+        Price:
```
Comment (nazarmatsevych) [inline src/index.html:36]: dont put text into `div` tag, create another `p` tag instead

File: src/styles/blocks/card.css [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  margin: 0 auto 4px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.product-card__code {
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
```
Comment (nazarmatsevych) [inline src/styles/blocks/card.css:25]: move color to variable

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
GJ, just small suggestions

File: src/index.html [hunk 4]
```diff
@@ -1,13 +1,43 @@
...
+        </div>
+
+        <p>Reviews: 5</p>
+      </div>
+
+      <p class="product-card__price-text">Price:</p>
+      
+      <p class="product-card__price-value">$2,199</p>
+
+      <a class="product-card__link" data-qa="hover">Buy</a>
```
Comment (witflash) [inline src/index.html:39]:
```suggestion
      <p class="product-card__price-text">Price:</p>
      <p class="product-card__price-value">$2,199</p>
      <a class="product-card__link" data-qa="hover">Buy</a>
```

There is no need to use empty lines between single-line siblings.

File: src/index.html [hunk 5]
```diff
@@ -1,13 +1,43 @@
...
+    </div>
+    
+  </body>
```
Comment (witflash) [inline src/index.html:42]:
```suggestion
    </div>
  </body>
```
There is no need to use an empty line between the child and the parent

File: src/styles/blocks/page.css [hunk 6]
```diff
@@ -0,0 +1,16 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.page {
+  --blue: #00acdc;
+  --gray: #616070;
```
Comment (witflash) [inline src/styles/blocks/page.css:7]: Better to name them `--color-primary` and `--color-secondary` for example

**Review by witflash — CHANGES_REQUESTED** [review]
Overall all is great!
Please, do some minor fixes.

File: src/index.html [hunk 7]
```diff
@@ -1,13 +1,39 @@
...
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./styles/index.css">
+  </head>
+  <body class="page-content">
+
+    <div class="product-card" data-qa="card">
```
Comment (Viktor-Kost) [inline src/index.html:12]: It's ok. But you can simplify the block naming and apply for other elements
```suggestion
    <div class="card" data-qa="card">
```

**Review by Viktor-Kost — APPROVED** [review]
GJ!

**Pull request closed without merging**